### PR TITLE
New weapons (Katana & Shiv)

### PR DESCRIPTION
**Pull request by DatBoiWeep**
- Added Katana & Shiv
- Added Katana to Blackmarkets with stock depending on location
- Shiv is only retrievable at the prison with random drops
- Added injuries for Katana & Shiv
- Fixed duplicate attributes for some black-market items
- Added images for new weapons
- Moved custom weapons to a new folder called [custom_weapons]

### CODE REVIEW

**Comment by ChangyBoi** [comment]
Moved blackmarkets to new positions

lowered the distance you see the prompt at

File: resources/usa_blackmarket/cl_blackmarket.lua [hunk 1]
```diff
@@ -131,7 +131,7 @@ Citizen.CreateThread(function()
       ------------------
       for k, v in pairs(markets) do
         local x, y, z = table.unpack(markets[k]['coords'])
-        DrawText3D(x, y, z, (markets[k]['3dTextDistance'] or 15), '[E] - Black Market')
+        DrawText3D(x, y, z, (markets[k]['3dTextDistance'] or 2), '[E] - Black Market')
```
Comment (minipunch) [inline resources/usa_blackmarket/cl_blackmarket.lua:134]:
```suggestion
        DrawText3D(x, y, z, (markets[k]['3dTextDistance'] or 5), '[E] - Black Market')
```

File: resources/usa_blackmarket/sv_blackmarket.lua [hunk 2]
```diff
@@ -3,7 +3,7 @@ math.random(); math.random(); -- prepare random price generator
 
 local markets = {
   ['marketA'] = {
-    ['coords'] = {-607.15, -1634.53, 33.05}, -- los santos
+    ['coords'] = {-424.69351196289, 179.0447845459, 80.596160888672}, -- los santos
```
Comment (minipunch) [inline resources/usa_blackmarket/sv_blackmarket.lua:6]: Changw

**Review by minipunch — APPROVED** [review]

File: resources/usa_blackmarket/cl_blackmarket.lua (hunk 1 above)
Comment (DatBoiWeep) [reply resources/usa_blackmarket/cl_blackmarket.lua:134]: Ill make another PR for this lol

**Pull request merged**